topics/elevate-users.md

```diff
@@ -26,6 +26,7 @@ This view contains all users who have logged in to Talis Elevate, and includes t
 | `login_count` | `INTEGER` | The number of times this user has logged in | `64` |
 | `last_login` | `TIMESTAMP` | Time the user last logged in. Timezone is UTC | `2016-01-12 00:00:00.0` |
 | `inst` | `VARCHAR(32)` | The tenant short code | `broadminster` |
+| `createdVia` | `VARCHAR(32)` | The method by which the user was created | `colleagueInvite` |
```

Comment:
maybe one for @timhodson but given this can only have 3 possible values maybe that should be documented here as well?

Reply:
+1. I'd vote for including the possible values in the description. If not obvious from the values themselves, then an explanation of the circumstances in which that value is seen would also be useful.

Reply:
I have updated the description